src/refract/element.es6

```diff
@@ -0,0 +1,92 @@
+/*
+ * Base elements for working with refract shorthand, which is composed of
+ * tuples with the following values:
+ *
+ * [name, metadata, attributes, contents]
+ *
+ * For example:
+ *
+ * ['resource', {'title': 'Question'}, {'href': '/questions'},
+ *   ['transition', {}, {}, ...],
+ *   ...
+ * ]
+ *
+ * https://github.com/refractproject/refract-spec
+ */
+const rawElement = Symbol('element');
+
+export default class RefractElement {
```

Comment:
@danielgtaylor I have plans to support everything in the Refract spec, but two things have been holding me back.

1. The spec has changed since starting minim, and it needs some work to get it up to speed (hence no support for metadata and so on)
1. I only get half a day per week to work on this, so I will be slow

With that said, if you want to update it and open PRs to it, that's totally fine. I had hoped to be able to convert to ES6 or vanilla JS.